### PR TITLE
Fix: description to get to "Add User Key"

### LINKED ISSUE #4576: Instructions to add a GitHub/Bitbucket user key don't reflect the new UI layout
# Locations
https://circleci.com/docs/2.0/gh-bb-integration/#enable-your-project-to-check-out-additional-private-repositories

# Details
The instructions apply to the legacy UI. The _**Project Settings > Checkout SSH keys**_ page no longer exists in the new UI. Instructions should now reference:

_**Project Settings > SSH keys**_ page -> _**Checkout SSH Keys > User Key**_ section

### PR DESCRIPTION
**Pull request by teesloane**
Fixes #4576

### CODE REVIEW

File: jekyll/_cci2/gh-bb-integration.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ GitHub user key in addition to the deploy key because each deploy key is valid
 for only _one_ repository while a GitHub user key has access to _all_ of your
 GitHub repositories.
 
-Provide CircleCI with a GitHub user key on your project's
-**Project Settings > Checkout SSH keys** page.
+Provide CircleCI with a GitHub user key on your project's **Project Settings >
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:86]:
```suggestion
Provide CircleCI with a GitHub user key in your project's **Project Settings** >
```

File: jekyll/_cci2/gh-bb-integration.md [hunk 2]
```diff
@@ -83,8 +83,8 @@ GitHub user key in addition to the deploy key because each deploy key is valid
...
+SSH keys**. Scroll down the page to **User Key** and click **Add User Key**.
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:87]:
```suggestion
**SSH keys**. Scroll down the page to **User Key** and click **Add User Key**.
```

**Comment by rosieyohannan** [comment]
Screenshot 2020-07-24 at 16 34 05

Only issue is I don't see these same steps when I'm looking at a CCI project

**Comment by teesloane** [comment]
Good catch @rosieyohannan  - updaated it!

**Pull request merged**